src/modules/setup.ts

```diff
@@ -41,30 +43,30 @@ class ReplaceVariablesStream extends Transform {
     _transform(chunk: any, _: string, callback: TransformCallback) {
         let result = this._variables.substitute(chunk.toString());
         let noticeComment: string;
-        const notice = 'This file is maintained by velocitas CLI, do not modify manually. Change settings in .velocitas.json';
         const shebang = '#!/bin/bash';
         const xmlDeclarationRegExp = new RegExp(`\\<\\?xml\\s.*?\\s\\?\\>`);
+        const fileExt = extname(this._filename);
 
         if (this._firstChunk) {
-            if (['.txt'].includes(this._fileExt)) {
-                result = `${notice}\n${result}`;
-            } else if (['.md', '.html', '.htm', '.xml', '.tpl'].includes(this._fileExt)) {
-                noticeComment = `<!-- ${notice} -->`;
+            if (['.txt'].includes(fileExt)) {
+                result = `${NOTICE_COMMENT}\n${result}`;
+            } else if (['.md', '.html', '.htm', '.xml', '.tpl'].includes(fileExt)) {
+                noticeComment = `<!-- ${NOTICE_COMMENT} -->`;
                 const xmlDeclarationArray = xmlDeclarationRegExp.exec(result);
                 if (xmlDeclarationArray !== null && result.startsWith(xmlDeclarationArray[0])) {
                     result = this._injectNoticeAfterStartLine(result, xmlDeclarationArray[0], noticeComment);
                 } else {
                     result = `${noticeComment}\n${result}`;
                 }
-            } else if (['.yaml', '.yml', '.sh'].includes(this._fileExt)) {
-                noticeComment = `# ${notice}`;
+            } else if (['Dockerfile'].includes(this._filename) || ['.yaml', '.yml', '.sh', '.dockerfile'].includes(fileExt)) {
```

Comment:
with this we would not allow people to have for example something like
"Dockerfile-test"
"Dockerfile-prod"
etc.